server/src/main.js

```diff
@@ -81,6 +85,7 @@ app.use(
   '/graphiql',
   graphiqlExpress({
     endpointURL: GRAPHQL_PATH,
+    subscriptionsEndpoint: `ws://localhost:${GRAPHQL_PORT}${SUBSCRIPTIONS_PATH}`,
```

Comment:
I don't think this is going to work in prod

Reply:
we can try it I guess? does the web UI connect directly to this or does the graphiql backend connect to it? If it's the former it's not going to work.